src/components/Card/CategoryCard/CategoryCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+function CategoryCard({category,img}) {
+  return (
+      <>
```

Comment:
you don't need these fragments, I see you have followed same pattern in other files, they are useless in your case

Reply:
Thanks for pointing it to me. Removed unnecessary fragments.